### PR TITLE
gemma2 had wrong path to scheduler

### PR DESCRIPTION
**Pull request by felipemello1**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [x] fix a bug
- [ ] update tests and/or documentation
- [ ] other (please add here)

### CODE REVIEW

**Review by joecummings — APPROVED** [review]
whoops

**Comment by RdoubleA** [comment]
Let's go ahead and delete the deprecated one in modules for the next release? I believe it's been two releases

https://github.com/pytorch/torchtune/blame/main/torchtune/modules/lr_schedulers.py#L18
